functional/start_etcd

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+USER_ID=${SUDO_UID:-$(id -u)}
+HOME=$(getent passwd "${USER_ID}" | cut -d: -f6)
```

Comment:
I don't really understand all these machinations. Why do we need HOME? why do we need to ensure we're sudo and not the root user? can't you just put it in temp and run it from there?